### PR TITLE
Release 6.4.2

### PR DESCRIPTION
**Pull request by forevermatt**
### Security
- Update to PHP 7.4

### Fixed
- Clean up leftover apt data after install
- Update dependencies
- Re-add browser check for whether U2F is supported (i.e. not Safari)

### CODE REVIEW

**Review by Schparky — APPROVED** [review]

**Comment by forevermatt** [comment]
Waiting to merge this until we determine whether the included version of module-material is broken or working (re: U2F support)

**Comment by forevermatt** [comment]
There are still several dependencies that need to be updated to work with PHP 7.4 as well.

**Review by Baggerone — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,15 @@
-FROM silintl/php7:7.2
+FROM silintl/php7:7.4
 
-MAINTAINER Phillip Shipley <phillip.shipley@gmail.com>
+LABEL maintainer "Phillip Shipley <phillip.shipley@gmail.com>"
 
-ENV REFRESHED_AT 2020-04-16
+ENV REFRESHED_AT 2021-06-14
```
Comment (Baggerone) [inline Dockerfile:5]: Heh. It's not worth changing, but did we all miss that the date/month was wrong or is it based on when you started working on the change?  :-)

**Review by Baggerone — APPROVED** [review]

**Review by Schparky — APPROVED** [review]

**Review by fillup — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (forevermatt) [reply Dockerfile:5]: Sadly, that's probably when I started working on this change... over a month ago

**Pull request merged**